test/test_adjoint.py

```diff
@@ -259,3 +259,32 @@ def test_sequential_sum_product_adjoint(
         )
         expected_bwd = expected_bwds[operand]
         assert (actual_bwd_t - expected_bwd).abs().data.max() < 5e-3 * num_steps
+
+
+@pytest.mark.parametrize(
+    "test",
+    [
+        None,
+        xfail_param("same"),
+        xfail_param("empty"),
+        xfail_param("other"),
+        xfail_param("reduce"),
+    ],
+)
+def test_identity_adjoint(test):
+    x = random_tensor(OrderedDict(i=Bint[3]))
+
+    with AdjointTape() as tape:
+        y = 2 * x
+        if test == "same":
+            y = y(i="i")
+        elif test == "empty":
+            y = y()
+        elif test == "other":
+            y = y(j=0)
+        elif test == "reduce":
+            y = funsor.terms.Reduce(ops.add, y, frozenset())
```

Comment:
In fact, it seems that any atomic op that doesn't change the funsor doubles the adjoint value.